Cannot create file .../microdds_client/dds_topics.yaml

I was getting this error locally before seeing it the nightly: 
https://github.com/osrf/vehicle_gateway/actions/runs/4877726326/jobs/8702648341

```
2023-05-04T00:52:43.2238831Z Architecture:  amd64
2023-05-04T00:52:43.2238987Z ==> CPACK_INSTALL_PREFIX = @DEB_INSTALL_PREFIX@
2023-05-04T00:52:43.2239401Z make[3]: warning: -j0 forced in submake: resetting jobserver mode.
2023-05-04T00:52:43.2239518Z Architecture:  amd64
2023-05-04T00:52:43.2239669Z ==> CPACK_INSTALL_PREFIX = @DEB_INSTALL_PREFIX@
2023-05-04T00:52:43.2240226Z cp: cannot create regular file '/__w/vehicle_gateway/vehicle_gateway/ros_ws/build/px4_sim/px4-src/src/px4-firmware/src/modules/microdds_client/dds_topics.yaml': No such file or directory
2023-05-04T00:52:43.2240733Z gmake[2]: *** [CMakeFiles/px4-firmware.dir/build.make:106: px4-src/src/px4-firmware-stamp/px4-firmware-install] Error 1
2023-05-04T00:52:43.2241049Z gmake[1]: *** [CMakeFiles/Makefile2:137: CMakeFiles/px4-firmware.dir/all] Error 2
2023-05-04T00:52:43.2241177Z gmake: *** [Makefile:146: all] Error 2
```

It seems that the `microdds_client` folder no longer exists in PX4. I wonder where it went... (will be looking into it).

--- a/px4_sim/CMakeLists.txt
+++ b/px4_sim/CMakeLists.txt
@@ -22,7 +22,7 @@ externalproject_add(px4-firmware
       build/px4_sitl_default/etc
       ${CMAKE_INSTALL_PREFIX}/share/px4_sim &&
     cp ${CMAKE_CURRENT_LIST_DIR}/dds_topics.yaml
-      ${PROJECT_BINARY_DIR}/px4-src/src/px4-firmware/src/modules/microdds_client/dds_topics.yaml
+      ${PROJECT_BINARY_DIR}/px4-src/src/px4-firmware/src/modules/uxrce_dds_client/dds_topics.yaml
   BUILD_COMMAND ${MAKE_EXECUTABLE} px4_ros2 -j && make px4_sitl_default -j
   BUILD_IN_SOURCE ON
   CONFIGURE_COMMAND ""

